.github/scripts/check-release.sh

```diff
@@ -2,10 +2,16 @@
 
 # Checking if current tag matches the package version
 current_tag=$(echo $GITHUB_REF | cut -d '/' -f 3 | tr -d ' ',v)
-file_tag=$(grep '"version":' package.json | cut -d ':' -f 2- | tr -d ' ' | tr -d '"' | tr -d ',')
-if [ "$current_tag" != "$file_tag" ]; then
-  echo "Error: the current tag does not match the version in package file(s)."
-  echo "$current_tag vs $file_tag"
+
+package_file_tag=$(grep '"version":' package.json | cut -d ':' -f 2- | tr -d ' ' | tr -d '"' | tr -d ',')
+package_file_name='package.json'
+version_file_tag=$(grep "PACKAGE_VERSION =" src/package-version.ts | cut -d "=" -f 2- | tr -d " " | tr -d "'")
+version_file_name='src/package-version.ts'
+
+if [ "$current_tag" != "$package_file_tag" ] || [ "$current_tag" != "$version_file_tag"]; then
```

Comment:
```suggestion
if [ "$current_tag" != "$package_file_tag" ] || [ "$current_tag" != "$version_file_tag" ]; then
```

Reply:
<img width="428" alt="Capture d’écran 2022-06-30 à 13 36 24" src="https://user-images.githubusercontent.com/20380692/176667950-e8d2171f-7621-4d17-84ed-8a8582a53c6d.png">

Reply:
Weird, it works for me
```
Error: the current tag does not match the version in package file(s).
package.json: 0.28.0 vs 0.7.1
src/package-version.ts: 0.28.0 vs 0.7.1
```

Reply:
My example was checking in the case everything is working (with `GITHUB_REF=refs/tags/v0.7.1`)